### ISSUE TITLE
[BUG] Can't save masks as PNG/tif using Cellpose GUI

### ISSUE THREAD

**Issue by LSeu-994** [MouseLand/cellpose#842]
**My issue**

Hello, 
I want to use cellpose segmentation masks of my experiment's images to do morphological analysis. To do that i need the masks images as tif.  i can try as much as i want to do so via GUI i can save the masks as seg.npy but not as PNG/tif. The GUI terminal produce each time the "GUI_INFO: saving 2D masks to png" line but nothing is produced in the folder itself. as a matter of fact i can run any other save/save as option without problem.

**To reproduce in my config**

- Lauching GUI via anaconda venv terminal using "python -m cellpose"
- import my segmented image in the GUI
- once loaded, run the save masks as PNG\tif

**Terminal log before running the option on the GUI**

2024-01-08 15:34:32,158 [INFO]
cellpose version:       2.2.3
platform:               win32
python version:         3.10.4
torch version:          1.12.1
2024-01-08 15:34:32,501 [INFO] ** TORCH CUDA version installed and working. **

Thanks a lot for your help.

**Comment by carsen-stringer** [comment]
sorry I can't reproduce this in 2D or 3D, can you please try the latest version of the code? and maybe `pip install imagecodecs --upgrade`

**Comment by LSeu-994** [comment]
Thank you for your reply! 

I think I've found the source of my problem. I work on several platforms and servers at work.

 I was running Cellpose on TIFF images placed on a remote server. In this situation, as I said above, all options worked except "GUI_INFO: saving 2D masks to png".

I solved this problem by moving my images to my local storage.

So i might be a network issue, unrelated with cellpose itself.
